# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/d3d12/CHANGELOG.md
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/d3d12/CHANGELOG.md b/third_party/rust/d3d12/CHANGELOG.md
--- a/third_party/rust/d3d12/CHANGELOG.md
+++ b/third_party/rust/d3d12/CHANGELOG.md
@@ -1,20 +1,24 @@
-# Change Log
-
-## v0.4.0 (2021-04-29)
-  - update `libloading` to 0.7
-
-## v0.3.1 (2020-07-07)
-  - create shader from IL
-  - fix default doc target
-  - debug impl for root descriptors
-
-## v0.3.0 (2019-11-01)
-  - resource transitions
-  - dynamic library loading
-
-## v0.2.2 (2019-10-04)
-  - add `D3DHeap`
-  - add root descriptor
-
-## v0.1.0 (2018-12-26)
-  - basic version
+# Change Log
+
+## v0.4.1 (2021-08-18)
+  - expose all indirect argument types
+  - expose methods for setting root constants
+
+## v0.4.0 (2021-04-29)
+  - update `libloading` to 0.7
+
+## v0.3.1 (2020-07-07)
+  - create shader from IL
+  - fix default doc target
+  - debug impl for root descriptors
+
+## v0.3.0 (2019-11-01)
+  - resource transitions
+  - dynamic library loading
+
+## v0.2.2 (2019-10-04)
+  - add `D3DHeap`
+  - add root descriptor
+
+## v0.1.0 (2018-12-26)
+  - basic version